src/librustc_trans/debuginfo/mod.rs

```diff
@@ -168,7 +168,7 @@ pub fn finalize(cx: &CrateContext) {
     }
 
     debug!("finalize");
-    let _ = compile_unit_metadata(cx);
+    //let _ = compile_unit_metadata(cx);
```

Comment:
I think maybe yeah? I'm not actually sure what changed here, it looks like just a bit of shuffling around to me...